app/src/main/java/com/wireguard/android/util/Extensions.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright © 2020 WireGuard LLC. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.wireguard.android.util
+
+import android.content.Context
+import android.util.TypedValue
+import androidx.annotation.AttrRes
+
+fun Context.resolveAttribute(@AttrRes attrRes: Int): Int {
+    val typedValue = TypedValue()
+    theme.resolveAttribute(attrRes, typedValue, true)
+    return typedValue.data
+}
```

Comment:
Is this a normal thing to have around? If so, why isn't it part of standard libraries?

Reply:
For resolving attributes from themes, totally. Not really sure why Google doesn't provide a helper function /shrug